contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -2064,12 +2064,19 @@ smallint_radians(PG_FUNCTION_ARGS)
 Datum
 bigint_power(PG_FUNCTION_ARGS)
 {
-	int64		arg1 = PG_GETARG_INT64(0);
-	Numeric		arg2 = PG_GETARG_NUMERIC(1);
+	int64		arg1;
+	Numeric		arg2;
 	int64 result;
 	Numeric		arg1_numeric,
 				result_numeric;
+	
+	/* when num or power is NULL, return NULL */
+
+	if (PG_ARGISNULL(0) | PG_ARGISNULL(1))
```

Comment:
Instead of checking NULL for each arguments in C function, we can direclty make sys.power function as `STRICT`.

Reply:
Addressed